oars_gb_pkg/helpers/path_planning/short_course_planner.py

```diff
@@ -0,0 +1,145 @@
+'''
+This program implements the short course routing algorithm described in Section 5.1 of Roland Stelzer's Autonomous
+Sailboat Navigation paper. This algorithm uses a polar efficiency function to determine the efficiency of all possible
+angles that a boat can turn, and then compares the efficiencies to determine the optimal boat heading.
+
+Read the paper here: https://www.dora.dmu.ac.uk/bitstream/handle/2086/7364/thesis-optimized-300dpi.pdf
+'''
+import numpy as np
+from turtle import *
```

Comment:
Since `turtle` and `time` are only used if this file is run directly by a Python interpreter, you really only need to run this if `__name__ == '__main__'. So you can move these lines to that section of the file below (just add them right after the check, before the rest of the code down there).